addons/test_mail/tests/test_message_post.py

```diff
@@ -387,3 +387,13 @@ def test_post_post_w_template(self):
             subject='About %s' % test_record.name,
             body_content=test_record.name,
             attachments=[('first.txt', b'My first attachment', 'text/plain'), ('second.txt', b'My second attachment', 'text/plain')])
+
+    @mute_logger('odoo.addons.mail.models.mail_mail')
+    def test_post_email_with_multiline_subject(self):
```

Comment:
Could you put it after ``test_post_answer`` ? that way even if currently some methods are not correclty ordered, we try to somehow keep an alphabetical ordering. Rationale is that tests are executed by alphabetical order, so having them in the same order in file helps debugging :) . Thanks !